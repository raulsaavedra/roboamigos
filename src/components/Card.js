import React from "react";


const Card = ({id, name, email}) => {
	return (
		<div className="bg-light-green pa3 ma2 tc dib bw2 shadow-5 grow">
			<img src={`https://robohash.org/${id}hehe`} width="200" height="200" alt="robots"/>
			<div>
			<h2>{name}</h2>
			<p>{email}</p>
			</div>
		</div>
	);
};

export default Card;
