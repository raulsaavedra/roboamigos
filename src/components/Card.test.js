import { shallowToJson  } from 'enzyme-to-json';
import {shallow} from 'enzyme';
import React from 'react';
import Card from './Card';

it('expect to render card component', () => {
	const card = shallow(<Card />);
	expect(shallowToJson(card)).toMatchSnapshot();
})