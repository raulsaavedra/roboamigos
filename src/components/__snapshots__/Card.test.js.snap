// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="bg-light-green pa3 ma2 tc dib bw2 shadow-5 grow"
>
  <img
    alt="robots"
    height="200"
    src="https://robohash.org/undefinedhehe"
    width="200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
